health-check/src/health_check/config.py

```diff
@@ -48,6 +51,10 @@ def get_config_dir_path(component: str) -> str:
     return os.path.join(CONFIG_DIR, component)
 
 
+def get_generated_config_dir_path(component: str) -> str:
```

Comment:
So we now have `get_generated_config_dir_path` and `get_config_dir_path`; the former for configs that we have to generate because of user input, the later for configs that we simply "hardcode".

Would it make sense to unify these two? Not sure tbh... It's a bit confusing as is right now, not sure how easy/difficult it is to always copy the `loki` config dir into `~/.local/share/health_check` dir.

Reply:
Hmm, yes indeed. I think we can improve this. I'm pushing an extra commit here for this